File: C/abc030A.c
#include <stdio.h>
 
int main(void)
{
    int a1, a2, b1, b2; 
    double a, b;
 
    scanf("%d%d%d%d", &a1, &a2, &b1, &b2);
    a = a2 * 1.0 / a1; 
    b = b2 * 1.0 / b1; 
 
    if(a < b){
        printf("AOKI\n");
    }
    if(a == b){
        printf("DRAW\n");
    }
    if(a > b){
        printf("TAKAHASHI\n");
    }
}

File: Java/abc030A.java
import java.util.Scanner;

class Main{
  public static void main(String args[]){
    Scanner scan = new Scanner(System.in);
    String a = scan.nextLine();
    String[] s = a.split(" ", 0);
    double b = Double.parseDouble(s[0]);
    double c = Double.parseDouble(s[1]);
    double d = Double.parseDouble(s[2]);
    double e = Double.parseDouble(s[3]);
    if(c/b < e/d){
      System.out.println("AOKI");
    }else if(c/b > e/d){
      System.out.println("TAKAHASHI");
    }else{
      System.out.println("DRAW");
    }
  }
}
